bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="sleepiq:dualBed">
+
+		<instruction-set targetVersion="1">
+			<add-channel id="todaySleepInBedSeconds">
```

Comment:
If I see understand the binding correctly, the channel shall be available for both channel groups, correct? I need to check if that is supported or if I need to add something more.

Reply:
Yes, that's correct.

Reply:
https://github.com/openhab/openhab-core/pull/3391 will bring the solution for this.